spec/systems/keyword_spec.rb

```diff
@@ -54,6 +64,21 @@
         click_button(I18n.t('keyword.upload'))
 
         expect(page).to have_content(I18n.t('keyword.upload_csv_successfully'))
+        expect(page).to have_selector('table')
```

Comment:
Let separate to another file @gutakk 1 system test file is 1 feature, don't merge them together, we will mix many features in the same system test file, that hard to maintain for later.

```
spec/systems/keywords/upload_keyword_spec.rb
spec/systems/keywords/view_list_keyword_spec.rb
```

Reply:
Fixed in 06b81f85cc2b03519422f74f502f71cdaf70f9c4